### PR TITLE
[DCOS-3014] dcos service --completed

### PR DESCRIPTION
**Pull request by mgummelt**

### CODE REVIEW

File: cli/dcoscli/service/main.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def _info():
 
 # TODO (mgummelt): support listing completed services as well.
 # blocked on framework shutdown.
```
Comment (tamarrow) [inline cli/dcoscli/service/main.py]: can delete this comment, no?

File: cli/tests/integrations/test_service.py [hunk 2]
```diff
@@ -121,6 +128,27 @@ def test_service_inactive(zk_znode):
...
+            cassandra_id = service['id']
+            break
+
+    service_shutdown(cassandra_id)
+
+    services = get_services(2)
+    assert not any(service['id'] == cassandra_id for service in services)
+
+    services = get_services(args=['completed'])
+    assert len(services) >= 3
```
Comment (tamarrow) [inline cli/tests/integrations/test_service.py]: why >=?

File: cli/dcoscli/service/main.py (hunk 1 above)
Comment (mgummelt) [inline cli/dcoscli/service/main.py]: yep. deleted

File: cli/tests/integrations/test_service.py (hunk 2 above)
Comment (mgummelt) [inline cli/tests/integrations/test_service.py]: includes services from other tests

**Comment by tamarrow** [comment]
lgtm when tests pass

**Pull request merged**